cmember.lisp

```diff
@@ -10,6 +10,19 @@
 (defmethod subctypep ((ct1 cmember) (ct2 cmember))
   (values (subsetp (cmember-members ct1) (cmember-members ct2)) t))
 
+(defun member-disjointp (cmember ctype)
```

Comment:
What do you think about making this function inline since its main purpose is to prevent duplication in the method bodies?

Reply:
subsetp is going to be way more of a bottleneck than call overhead, but inlining is harmless